### PR TITLE
11.0

### PR DESCRIPTION
**Pull request by vikas-vg**
fix things which aren't working:
    1. images aren't loading in website product page.
    2. image_url field ins't working.

### CODE REVIEW

File: website_multi_image_zoom/controllers/main.py [hunk 1]
```diff
@@ -10,6 +10,6 @@ class WebsiteSale(main.WebsiteSale):
                 methods=['POST'])
     def get_variant_images(self, **post):
         variant_images = request.env['product.image'].sudo(). \
-            search([('product_variant_id', '=', int(post.get('product_id')))])
+            search([('product_variant_id', 'in', int(post.get('product_id')))])
```
Comment (JayVora-SerpentCS) [inline website_multi_image_zoom/controllers/main.py]: the right end of domain is not list, so 'in' is not right.

Comment (vikas-vg) [reply website_multi_image_zoom/controllers/main.py]: I fixed this bug and also, some other bugs and a little feature update in my latest pull requests. Please review.

**Comment by JayVora-SerpentCS** [comment]
@hpatelserpentcs Please review

**Comment by hpatelserpentcs** [comment]
@JayVora-SerpentCS 
I will check it in my local system and update here. 
@sadavarti  Do you really want to Change field type in stable version ? (It will break existing Environment)

**Comment by vikas-vg** [comment]
Sorry, for late reply.
@hpatelserpentcs It isn't necessary to change the field type but I think there are some images which users want to add for multiple variants and that's why change of field type.
I really think it is a useful feature.

**Pull request merged**